src/components/SearchModal/ImportToken.tsx

```diff
@@ -33,6 +35,7 @@ interface ImportProps {
 export function ImportToken(props: ImportProps) {
   const { tokens, list, onBack, onDismiss, handleCurrencySelect } = props
```

Comment:
If tokens is always a list of one, replace it with `token: Token` in ImportProps and then fix it where those props are used. Otherwise, if it's possible for tokens to be plural, it should be supported correctly.

Reply:
@willhennessy I think, if it's never going to be plural, this is the best way forward.